db/migrate/20171113153232_create_batches.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBatches < ActiveRecord::Migration[5.1]
+  def change
+    create_table :batches do |t|
+    	t.string :track
```

Comment:
What will track represent? Like `Web Development`, `Digital Marketing` etc? If so, then it seems like using an integer value here, and adding a `enum` on the `Batch` model would be more appropriate for this.